_data/ph_authors.yml

```diff
@@ -1357,3 +1357,12 @@
     - editorial
     - proghist
   status: volunteer
+
+- name: Josir Cardoso Gomes
+  team: false
```

Comment:
in this case is team: true

Reply:
I understood that team was just for the main team. But I got it now.